namespaces/http.yml

```diff
@@ -0,0 +1,60 @@
+version: 2016.03.10.0
+
+defaults:
+  type: string
+  doc_values: true
+  index: not_analyzed
+
+namespace:
+  name: http
+  type: group
+  description: >
+    Attributes common to http requests.
+  fields:
+
+    - name: client_ip
+      type: ip
+      description: >
+        IP address of the remote client.
+      norms:
+        enabled: True
+
+    - name: request_method
+      type: string
+      description: >
+        HTTP request method (GET, POST, PUT, etc)
+      norms:
+        enabled: True
+
+    - name: request_path
+      type: string
+      description: >
+        HTTP request path
+      norms:
+        enabled: True
```

Comment:
I think we need norms disabled for not_analyzed fields (default is disabled for not_analyzed fields)